modules/tests/test/pipelines_test.go

```diff
@@ -1 +1,239 @@
 package test
+
+import (
+	"context"
+
+	. "github.com/kubevirt/kubevirt-tekton-tasks/modules/tests/test/constants"
+	"github.com/kubevirt/kubevirt-tekton-tasks/modules/tests/test/framework"
+	"github.com/kubevirt/kubevirt-tekton-tasks/modules/tests/test/runner"
+	"github.com/kubevirt/kubevirt-tekton-tasks/modules/tests/test/testconfigs"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/apis"
+)
+
+var _ = Describe("Pipelines tests", func() {
+	f := framework.NewFramework()
+	It("DV is created, disk-virt-sysprep, create dv, delete dvs", func() {
+		config := &testconfigs.PipelineTestConfig{
```

Comment:
Can we load pipelines from YAML to be able to test different pipelines? (e.g. `windows10-installer`?)

Reply:
Currently it is not needed. But it is a good idea for enhancement. https://github.com/kubevirt/kubevirt-tekton-tasks/issues/181